frontend/packages/policy-editor/src/utils/index.ts

```diff
@@ -251,7 +215,7 @@ export const convertSubjectStringToSubjectId = (subjectString: string): string =
 export const createNewSubjectFromSubjectString = (subjectString: string): PolicySubject => {
   const subjectId: string = convertSubjectStringToSubjectId(subjectString);
   return {
-    subjectId: subjectId,
+    subjectId: subjectId.toLowerCase(),
     subjectTitle: subjectId,
```

Comment:
Is the subjectTitle suppose to be the same as subjectId?

Reply:
I agree that it seems very confusing, but as I understand it, the frontend doesn't have all the necessary data to construct the object correctly at this time. Therefore, it's being set to subjectId. Here, one could have chosen to use a null or undefined value instead. I chose not to change this in this PR because there is a risk that other things could go wrong in the code, creating a new bug. There are many dependencies and prop-drilling involved. This is something we need to clean up in the next round when we have specific issues for what to do next with the Policy Editor.